sql/core/src/test/scala/org/apache/spark/sql/CountMinSketchAggQuerySuite.scala

```diff
@@ -61,4 +61,23 @@ class CountMinSketchAggQuerySuite extends QueryTest with SharedSparkSession {
 
     assert(sketch == reference)
   }
+
+  test("count-min sketch with named-arguments") {
```

Comment:
Shall we add some e2e tests on the function to `sql/core/src/test/resources/sql-tests/inputs/named-function-arguments.sql`?